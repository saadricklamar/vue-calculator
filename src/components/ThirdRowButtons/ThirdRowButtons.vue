<template>
  <section class="third-row-buttons">
    <button v-on:click="displayBtnValue($event)" class="four" value="4">4</button>
    <button v-on:click="displayBtnValue($event)" class="five" value="5">5</button>
    <button v-on:click="displayBtnValue($event)" class="six" value="6">6</button>
    <button v-on:click="subtract($event)" class="subtract" value="-">-</button>
  </section>
</template>

<script>
export default {
  name: "ThirdRowButtons",
  props: ["displayBtnValue", "subtract"]
};
</script>

<style>
.subtract {
  background-color: #42b983;
  color: white;
}
</style>