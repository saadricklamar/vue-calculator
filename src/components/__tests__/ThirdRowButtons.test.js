import { mount } from "@vue/test-utils";
import ThirdRowButtons from "../ThirdRowButtons/ThirdRowButtons";

describe("ThirdRowButtons", () => {
  const wrapper = mount(ThirdRowButtons);
  it("should match the snapshot", () => {
    expect(wrapper.isVueInstance()).toMatchSnapshot();
  });
});