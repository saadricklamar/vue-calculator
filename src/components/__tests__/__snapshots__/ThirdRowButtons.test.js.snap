// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThirdRowButtons should match the snapshot 1`] = `true`;
